Flat/src/ModulesLepPho.cc

```diff
@@ -183,15 +203,16 @@ void PandaAnalyzer::ComplicatedLeptons() {
     looseLeps.push_back(&ele);
     matchLeps.push_back(&ele);
     matchEles.push_back(&ele);
-    gt->nLooseElectron++;
+    // WARNING: The definition of "loose" here may not match your analysis definition of a loose electron for lepton multiplicity or jet cleaning considerations.
+    // It is the user's responsibility to make sure he is cutting on the correct multiplicity. Enough information is provided to do this downstream.
```

Comment:
To be clear, this only applies to ComplicatedLeptons right?

Reply:
yes